x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result.scss

```diff
@@ -0,0 +1,55 @@
+.appSearchResult {
+  overflow: hidden;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+
+  &__contentWrap {
+    width: 100%;
+    padding: $euiSize;
+    display: grid;
+    grid-template-rows: 1fr $euiSizeXS * 5;
```

Comment:
I actually don't think we need `display: grid` on `__contentWrap` - just on `__contentInner`. We could easily lose this and simply put a margin-top on `__hiddenFieldsIndicator` and get the same visual effect

```suggestion
```